development/image-syncer/README.md

```diff
@@ -9,7 +9,7 @@ It copies images **only when they are not present** in the target repo. That gua
 
 Syncing process steps:
 1. Pull image from source.
-2. Check if image name contains SHA256 digest.
+2. Check if image name contains the multi-platform SHA256 digest. To get this digest, run `docker run mplatform/manifest-tool inspect {your-image}:{image-version}` and copy the first Digest this command returns. For the non-multi-platform images supporting only amd64 architecture, use their amd64 digest and add the `amd64only: true` param. 
```

Comment:
```suggestion
2. Check if the image name contains the multi-platform SHA256 digest. To get this digest, run `docker run mplatform/manifest-tool inspect {your-image}:{image-version}` and copy the first digest this command returns. For the non-multi-platform images supporting only amd64 architecture, use their amd64 digest and add the `amd64only: true` param. 
```
Just wanted to make sure if "Digest" should be capitalized here. It is not in other parts of the paragraph.